scripts/installers/ubuntu-20.04-install.sh

```diff
@@ -248,7 +248,9 @@ function install_armui_service() {
 
 function launch_setup() {
     echo -e "${RED}Launching ArmUI first-time setup${NC}"
-    site_addr=$(sudo netstat -tlpn | awk '{ print $4 }' | grep ".*:${PORT}")
+    echo "Giving ArmUI a moment to start, standby..."
+    sleep 30
```

Comment:
Didn't I already add this ?

Reply:
I thought I added it as part of 2.6.9 but I guess I forgot to copy it over. I'm basing off of v2_devel as of like an hour ago too